java/src/main/java/ai/rapids/cudf/Schema.java

```diff
@@ -182,7 +182,7 @@ public boolean hasNestedChildren() {
     return false;
   }
 
-  int[] getFlattenedTypeIds() {
+  public int[] getFlattenedTypeIds() {
```

Comment:
Can you add some javadocs to these methods that are now public. If they are public at a minimum we have to document how they are flattened so that a user could make use of it.

Reply:
I've added Java docs for the changed functions. However, there are other similar public functions don't have Java docs thus I've also added docs for them for consistency.